code/search/fibonacci_search/fibonacci_search.swift

```diff
@@ -0,0 +1,70 @@
+/* Part of Cosmos by OpenGenus Foundation */
+
+//  fibonacci_search.swift
+//  Created by erwolfe on 11/17/2017
+
+/**
+ Fibonacci Search looks through an array for an element and returns it if its found otherwise -1 is returned.
+
+ - parameters:
+     - numberArray: An array of ints that is being traversed to find an element.
+     - x: An int that is been looked for in the array.
+     - n: The size of the of the array.
+
+ - returns:
+ An int that is the index of the element in the array unless its not found and then -1 is returned.
+ */
+public func fibonacciSearch(_ numberArray: [Int],_ x: Int, _ n: Int) -> Int {
```

Comment:
Although the coding style is not said put one space after each parameters, but I think you want put it...? so let it be rule. (don't forgot add this to coding style if you want.)

Reply:
So you want this?

```swift
public func fibonacciSearch(_ numberArray: [Int], _ x: Int, _ n: Int) -> Int {
```

Reply:
yes.